packages/icons/plugins/svg-replace-plugin.js

```diff
@@ -0,0 +1,26 @@
+//Replace <svg> tag with <Svg>
+const replaceSvgPlugin = ({ types: t }) => {
+  const replaceSvg = {
+    JSXElement(path) {
+      if (!path.get('openingElement.name').isJSXIdentifier({ name: 'svg' })) {
+        return
+      }
+      const openingElementName = path.get('openingElement.name')
```

Comment:
Also logging in upcoming issue:

Will the path always get `openingElement.name`? If not, maybe a conditional check to be safe w. `.replaceWith` 👇?